docs/updating.md

```diff
@@ -0,0 +1,124 @@
+# Updating & releasing Accompanist
+
+This doc is mostly for maintainers.
+
+## New features & bugfixes
+All new features should be uploaded as PRs against the `main` branch. 
+
+Once merged into `main`, they should be merged into the `snapshot` branch:
```

Comment:
Can this be automated with an action?

Reply:
Too risky, the Compose APIs might have changed in the snapshot.